_posts/blog/2017-02-23-security-performance-implications.md

```diff
@@ -0,0 +1,158 @@
+|Master Node|i2.xlarge|4 cores, 30.5 GB RAM, 1x800GB SSD|2|
+|Admin Node|m4.xlarge|4 cores, 16 GB RAM|1|
+|Timely Node|m4.2xlarge|8 cores, 32 GB RAM|1|
+
+
+Table 2 – AWS Instance Types, Role, Details, and Quantities
+
+
+## Results
+
+The median, max, and min of the milliseconds elapsed
+time of all iterations for each test is displayed below. The percentage change
+columns compare the Median, Max, and Min respectively from the No
+Security level to each security configuration (e.g. No Security Median
+vs. Auth-Int Median, No Security Max vs. Auth-Int Max).
+
+
+{: #results .table }
+| Security Level |  Median  |  Standard Deviation  |  Max   |    Min   |    % Change (nosec Median vs. Median) |  % Change (nosec Max vs. Max) |  % Change (nosec Min vs. Min)  | Delta from Previous Level (Median)|
+| ---------------- |---------: |---------:|----------:| ---------:| ------------------------------------: |------------------------------:| ------------------------------:| ------------------------------------:|
+| No security  |    7829394  |  139340  | 8143035|   7764309  | 0.00%    |                            0.00%    |                      0.00%    |                      0.00%|
+|ssl        |      8292760  |     87012   |  8464060  | 8204955 |  5.92% |                               3.94%      |                    5.68%           |               5.92%|
+| auth        |     8859552 |    134109    | 9047971|   8657618  | 13.16%    |                           11.11%           |              11.51%            |             6.83%|
+| auth-int     |    9500737 |    155968    |   9753424  | 9282371  | 21.34%       |                        19.78%               |          19.55%         |                7.24%|
+|auth-conf    |    9479635  |    170823   |  9776580  | 9282189  | 21.08%          |                     20.06%        |                 19.55%       |                  -0.22%|
+
+Table 2 – Summarized Time Elapsed for Each Security Level
+
+
+## Plots
+
+Below are some snapshots of *stats.out elements via Grafana that were inserted
+into Timely with the same relative start time.
+
+### TABLE\_RECS (Down sampling 1m, aggregate=avg)
+
+![]({{ site.baseurl}}/images/blog/201702_security/tableRecs.png){:width="800px"}
+
+### TOTAL\_INGEST (Down sampling 5m, aggregate=avg)
+
+![]({{ site.baseurl}}/images/blog/201702_security/totalIngest.png){:width="800px"}
+
+### AVG\_FILES/TABLET (Down sampling 1m, aggregate=avg)
+
+![]({{ site.baseurl}}/images/blog/201702_security/avgFilesTab.png){:width="800px"}
+
+### ACCUMULO\_FILES (Down sampling, 1m, aggregate=avg)
+
+![]({{ site.baseurl}}/images/blog/201702_security/accumuloFiles.png){:width="800px"}
+
+
+As can be seen in the plots above, the different security settings have 
+relatively consistent, discernable median run characteristics.  The big
+dip in each TOTAL_INGEST coincides with a large number of major
+compactions, a rate decrease for TABLE_RECS, and a decrease in 
+AVG_FILES/TABLET. 
```

Comment:
Can you use the "human words/phrases" to describe the results here instead of the graph labels (or just include the label in parens)? I don't think that the labels aren't clear enough for the average user to understand what is being displayed.